Dockerfile

```diff
@@ -1,10 +1,41 @@
+# Container build stage layout are as followed:
+#      debian         node
+#        |             |
+#       base           |
+#     /      \         |
+#     |   backend  frontend
+#     |         \   /
+#    dev        final
+#
+# base: necessary base runtime and build dependencies like python
+# backend: installed backend src-code
+# fronted: installed frontend src-code
+# dev: configuration links but no code since that gets mounted
```

Comment:
How do I mount my source code? Or does this happen automatically?

Reply:
It's still docker so mounts happen when the container starts. In this case, it happens here:
 https://github.com/TheNerdful8/Lecture2Gether/blob/92c91f67cce4b5229b47c88e4930b1ad108e3a7c/dev_up.sh#L12